src/components/Currencies.js

```diff
@@ -1,34 +1,94 @@
+/*
```

Comment:
I added Radium to this component as well.
I decided to add CSS grid here in place of the custom grid I was using before.